cmd/server/http/http.go

```diff
@@ -140,22 +138,3 @@ func trace(tracer tracing.Tracer) func(http.Handler) http.Handler {
 		})
 	}
 }
-
-// authenticate authenticates the handler against a Bearer token.
-//
-// If authentication fails a 401 Unauthorized HTTP status is returned with an
-// ErrorResponse body.
-func authenticate(token string) func(http.Handler) http.Handler {
```

Comment:
It would be great to allow both authentication mechanisms for a while to test the new login flow before forcing everyone to use it.